src/surface.lisp

```diff
@@ -11,8 +11,12 @@
   (c-ref surface sdl2-ffi:sdl-surface :pixels))
 
 (defun surface-format (surface)
+  (c-ref surface sdl2-ffi:sdl-surface :format))
+
+(defun surface-format-format (surface)
   (enum-key '(:enum (sdl-pixel-format))
-            (c-ref surface sdl2-ffi:sdl-surface :format :format)))
+            (c-ref (c-ref surface sdl2-ffi::sdl-surface :format)
+                   sdl2-ffi::sdl-pixel-format :format)))
 
```

Comment:
OK. And, sadly not.. a few things fixed in the past couple days, this being (headdeskingly) one of them.